backend/app/data_outputs/router.py

```diff
@@ -0,0 +1,39 @@
+from uuid import UUID
+
+from fastapi import APIRouter, Depends
+from sqlalchemy.orm import Session
+
+from app.data_outputs.schema import DataOutput, DataOutputCreate
+from app.data_outputs.service import DataOutputService
+from app.database.database import get_db_session
+
+router = APIRouter(prefix="/data_outputs", tags=["data_outputs"])
+
+
+@router.get("")
+def get_data_outputs(db: Session = Depends(get_db_session)) -> list[DataOutput]:
+    return DataOutputService().get_data_outputs(db)
```

Comment:
```suggestion
    return DataOutputService(db).get_data_outputs()
```
Since you already instantiate an object I suggest passing in 'db' to `init` method. In such a case if you will need to use db in multiple methods you can use self.db instead of passing it as an argument to another method